envoy/upstream/upstream.h

```diff
@@ -585,14 +612,15 @@ class PrioritySet {
   COUNTER(lb_zone_routing_all_directly)                                                            \
   COUNTER(lb_zone_routing_cross_zone)                                                              \
   COUNTER(lb_zone_routing_sampled)                                                                 \
-  COUNTER(membership_change)                                                                       \
+  GAUGE(lb_subsets_active, Accumulate)
+
+/**
+ * All cluster stats. @see stats_macros.h
+ */
+#define ALL_CLUSTER_STATS(COUNTER, GAUGE, HISTOGRAM, TEXT_READOUT, STATNAME)                       \
```

Comment:
I think maybe we should rename this macro and related call-sites because this does not represent all the stats anymore.

Reply:
I did that in another CL. See https://github.com/envoyproxy/envoy/pull/23921/files

Reply:
can we focus on just making this one coherent? and getting it in?

Reply:
done.